gitkraken-client/git-flow.md

```diff
@@ -67,3 +67,18 @@ Hotfixes are for quickly pushing out a change to your production branch.  Common
 When finishing a hotfix, GitKraken Client will merge the changes into both `master` and `develop`.
 
 <img src="/wp-content/uploads//finish-hotfix.gif" class="img-bordered img-responsive center">
+
+### Tag
+
+Tags are used to mark a specific point in the repository's history.  They are often used to mark a release.
+
+Tags can be created from the Gitflow menu, or from the command line. When creating a tag from the Gitflow menu, GitKraken Client will create a tag with the same name as the branch. For example, if you create a tag from a `release/1.0.0` branch, GitKraken Client will create a tag named `1.0.0`. Additionally, you can add a tag message when fishing a branch. This message will be added to the tag.
+
+<img src="/wp-content/uploads//finish-release-tag.gif" class="img-bordered img-responsive center">
+
+<img src="/wp-content/uploads//tag-message.png" class="img-bordered img-responsive center">
+
+In Preferences > Gitflow, you can set a tag prefix.  This prefix will be added to the tag name when creating a tag from the Gitflow menu.  For example, if you set the tag prefix to `v`, and create a tag from a `release/1.0.0` branch, GitKraken Client will create a tag named `v1.0.0`.
```

Comment:
We mostly are using `` to note navigating preferences i.e. `Preferences > Gitflow`